### PR TITLE
Code splitting

### PR DESCRIPTION
**Pull request by akshay1027**
## Related Issue 

Implemented code splitting and improved application s performance from 29 to 62 and accessibility from 86 to 100 ( in google dev tools performance metric)

Closes: #189 

### Describe the changes you've made

Since the home takes a 100vh, we can take some more time to load the footer in home and other routes, so implemented lazy loading! Also made the 404 page lazy-loaded

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
Any additional information you want to give

### CODE REVIEW

**Comment by akshay1027** [comment]
### before code splitting:  

  
 
### after code splitting:

**Comment by akshay1027** [comment]
@Namanl2001 please let me know about the changes!

File: src/App.js [hunk 1]
```diff
@@ -2,26 +2,53 @@ import React from 'react';
...
           <Switch>
             <Route exact path='/' component={Home} />
+
             <Route path='*' component={E404} />
           </Switch>
         </React.Suspense>
       </Router>
-      <Footer />
+      <React.Suspense
+        fallback={
```
Comment (Namanl2001) [inline src/App.js:37]: What this fallback is for?
Is it for the time period until the lazy components are shown?

**Review by Namanl2001 — APPROVED** [review]
Just a question. Else LGTM. Thanks for your great work! 👍

File: src/App.js (hunk 1 above)
Comment (Namanl2001) [reply src/App.js:37]: @akshay1027 I guess you missed the question

**Review by akshay1027 — COMMENTED** [review]
Oh okay so you asked about fallback itself is it, yes    
It is for the time period until the lazy components are shown

File: src/App.js (hunk 1 above)
Comment (akshay1027) [reply src/App.js:37]: Oh okay so you asked about fallback itself is it, yes
It is for the time period until the lazy components are shown

**Pull request merged**